apps/namadillo/src/App/AccountOverview/ShieldedAssetsOverview.tsx

```diff
@@ -0,0 +1,62 @@
+import { ActionButton, Panel } from "@namada/components";
+import { MaspSyncCover } from "App/Common/MaspSyncCover";
+import { ShieldedTransferIcon } from "App/Icons/ShieldedTransferIcon";
+import { UnshieldedTransferIcon } from "App/Icons/UnshieldedTransferIcon";
+import { ShieldedAssetTable } from "App/Masp/ShieldedAssetTable";
+import clsx from "clsx";
+import { useAmountsInFiat } from "hooks/useAmountsInFiat";
+import { useRequiresNewShieldedSync } from "hooks/useRequiresNewShieldedSync";
+import { EstimateShieldingRewardsCard } from "./EstimateShieldingRewardsCard";
+import { TotalBalanceCard } from "./TotalBalanceCard";
+
+export const ShieldedAssetsOverview = (): JSX.Element => {
+  const { shieldedAmountInFiat } = useAmountsInFiat();
+  const textContainerClassList = `flex h-full gap-1 items-center justify-center`;
+  const requiresNewShieldedSync = useRequiresNewShieldedSync();
+
+  return (
+    <Panel className="relative z-10 px-6 border border-yellow rounded-t-none -mt-px">
+      <div className="flex items-center gap-16 mt-4">
+        <TotalBalanceCard
+          balanceInFiat={shieldedAmountInFiat}
+          isShielded={true}
+          footerButtons={
+            <>
+              <ActionButton
+                outlineColor="yellow"
+                size="xs"
+                className="w-auto px-4"
+              >
+                <span className={clsx(textContainerClassList)}>
+                  Shielded Transfer
```

Comment:
<img width="340" alt="Screenshot 2025-04-01 at 5 43 40 PM" src="https://github.com/user-attachments/assets/7f0a164b-ae03-4a3d-9216-cbe5b7a3ab77" />

Neither of these buttons work for me. I def have balance to `Unshield`